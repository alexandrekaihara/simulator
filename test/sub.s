LOAD TEN
SUB MINUSONE
SUB ELEVEN
JMPZ ISZERO
OUTPUT MINUSONE
STOP
ISZERO: LOAD FIVE
SUB TEN
SUB MINUSFIVE
JMPZ ISZERO2
OUTPUT MINUSONE
STOP
ISZERO2: OUTPUT ONE
STOP
TEN: CONST 10
ELEVEN: CONST 11
MINUSONE: CONST -1
MINUSNINE: CONST -9
FIVE: CONST 5
MINUSFIVE: CONST -5
ONE: CONST 1