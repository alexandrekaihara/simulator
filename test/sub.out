10 25 2 27 2 26 8 11 13 27 14 10 29 2 25 2 30 8 22 13 27 14 13 31 14 10 11 -1 -9 5 -5 1